plugin/evm/vm.go

```diff
@@ -127,8 +129,25 @@ const (
 	bytesToIDCacheSize  = 5 * units.MiB
 
 	targetAtomicTxsSize = 40 * units.KiB
+
+	// p2p app protocols
+	ethTxGossipProtocol    = 0x0
+	atomicTxGossipProtocol = 0x1
+
+	// gossip constants
+	txGossipBloomMaxFilledRatio    = 0.75
+	txGossipBloomMaxItems          = 8 * 1024
+	txGossipBloomFalsePositiveRate = 0.01
+	maxValidatorSetStaleness       = time.Minute
+	throttlingPeriod               = 5 * time.Second
+	throttlingLimit                = 5
```

Comment:
So, this is specifying that we allow ~1 request per second from peers. However, we poll once every 10 seconds from 10 random validators... Seems like we could be much more strict on the rate limiting.

Reply:
Yeah we can definitely line these up to be consistent, like 1 call per 10 seconds.